holiday-calendar-eur/src/main/java/com/github/davejoyce/calendar/impl/HolidayCalendarServiceCH.java

```diff
@@ -0,0 +1,43 @@
+package com.github.davejoyce.calendar.impl;
+
+import com.github.davejoyce.calendar.Holiday;
+import com.github.davejoyce.calendar.HolidayCalendar;
+import com.github.davejoyce.calendar.HolidayCalendarService;
+
+import java.time.Month;
+
+import static com.github.davejoyce.calendar.HolidayCalendar.STANDARD_WEEKEND;
+
+/**
+ * Service for provision of Switzerland national holiday calendar.
+ *
+ * @author <a href="mailto:dave@osframework.org">Dave Joyce</a>
+ */
+public class HolidayCalendarServiceCH implements HolidayCalendarService {
+
+    private static final String CODE = "CH";
+    private static final String NAME = "Switzerland National Holidays";
+
+    @Override
+    public boolean isProvided(String code) {
+        return CODE.equalsIgnoreCase(code);
+    }
+
+    @Override
+    public HolidayCalendar getHolidayCalendar() {
+        final Holiday swissNationalDay = Holiday.builder()
+                                                .name("Swiss National Day")
```

Comment:
Why just Swiss National Day?  Switzerland also does Christmas Day, New Year's Day, Good Friday and Ascension Day by the looks of it.

Reply:
@battleblow I think we should discuss national vs traditional holiday inclusion, as well as regional holiday inclusion. This could be a union of sets, a configurable feature flag, or something else. Regardless, we should agree and then prioritize it.